### ISSUE TITLE
Allow for latest build on https://flipper-xtre.me/update/

### ISSUE THREAD

**Issue by ALEEF02** [ClaraCrazy/Flipper-Xtreme#585]
### Describe the enhancement you're suggesting.

Good afternoon,

I would appreciate it if there was a CI/CD pipeline for XFW, so that I could install the latest development build/push from the website without having to pull the repo, update submodules, fix my environment, and then finally build and upload via the fbt. This should be a relatively small implementation since there exists some CI/CD already.

**Comment by dropalltables** [comment]
This actually sounds quite interesting!

**Comment by dropalltables** [comment]
but @ALEEF02 it exists on github actions
https://github.com/Flipper-XFW/Xtreme-Firmware/actions/workflows/build.yml

**Comment by ALEEF02** [comment]
Yes, I understand that I can download the builds from GitHub and then use qFlipper, however, I do believe a website integration so that it can be uploaded straight through the webserialAPI,

> but @ALEEF02 it exists on github actions https://github.com/Flipper-XFW/Xtreme-Firmware/actions/workflows/build.yml

**Comment by WatskeBart** [comment]
> Yes, I understand that I can download the builds from GitHub and then use qFlipper, however, I do believe a website integration so that it can be uploaded straight through the webserialAPI,
> 
> > but @ALEEF02 it exists on github actions https://github.com/Flipper-XFW/Xtreme-Firmware/actions/workflows/build.yml

You can use [https://lab.flipper.net/](https://lab.flipper.net/) to upload the *.tgz from the workflow build.

Of course it would be great when [https://flipper-xtre.me/update/](https://flipper-xtre.me/update/) offers a feature to upload from a file, or directly from a workflow.
